iOS_SDK/OneSignalSDK/UnitTests/UnitTests.m

```diff
@@ -1401,8 +1316,8 @@ -(void)assertLocalNotification:(NSDictionary*)userInfo {
 }
 
 // Testing iOS 8 - with os_data aps payload format
```

Comment:
Comment should be `// Testing iOS 9` now?

Reply:
Fixed